money-aws-java-sdk/src/test/scala/com/comcast/money/java/AmazonWebServiceClientMoneyTraceRequestHandlerTest.java

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright 2012-2015 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.comcast.money.java;
+
+import static org.junit.Assert.assertNotNull;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import com.amazonaws.Request;
+import com.amazonaws.Response;
+import com.comcast.money.core.SpanId;
+import com.comcast.money.internal.SpanLocal;
+import com.comcast.money.japi.JMoney;
+
+import scala.Option;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({JMoney.class, SpanLocal.class, Request.class, Response.class})
+public class AmazonWebServiceClientMoneyTraceRequestHandlerTest {
+
+    private AmazonWebServiceClientMoneyTraceRequestHandler handler;
+
+    private final String clientName = "clientName";
+
+    @Before
+    public void setUp() {
+
+        handler = new AmazonWebServiceClientMoneyTraceRequestHandler(clientName);
+    }
+
+    @Test
+    public void testAmazonWebServiceClientMoneyTraceRequestHandler() {
+
+        assertNotNull(handler);
+    }
+
+    @Test
+    public void testBeforeRequestRequestOfQ_moneyNotEnabled() {
+
+        PowerMockito.mockStatic(JMoney.class, SpanLocal.class);
```

Comment:
Not sure how MoneyHelper wrapper will help since the methods we are trying to mock are scope=static in both those classes. I am not a big fan of powermock as well, it is a weapon of last resort to mock static method calls or final classes.